extendables/music.js

```diff
@@ -0,0 +1,13 @@
+const { Extendable } = require("klasa");
```

Comment:
no... music should be a property of guild not a getter

Reply:
Done

Reply:
not done, you made it a property... you cant do instanceof properties with extendables please use Discord.js#Structures.extend